tests/testnet.spec.js

```diff
@@ -99,6 +99,7 @@ describe('Testing generic actions', () => {
   const newFioAddress = generateTestingFioAddress(newFioDomain)
   const privateKeyExample = '5Kbb37EAqQgZ9vWUHoPiC2uXYhyGSFNbL6oiDp24Ea1ADxV1qnu'
   const publicKeyExample = 'FIO5kJKNHwctcfUM5XZyiWSqSTM5HTzznJP9F3ZdbhaQAHEVq575o'
+  const newPublicKey = 'FIO7isxEua78KPVbGzKemH4nj2bWE52gqj8Hkac3tc7jKNvpfWzYS'
```

Comment:
don't know should we leave this key here, maybe we could make it empty and rename it to `pubKeyForTransfer`, or just use `publicKey2 `

Reply:
I like this approach. Will make corrections.